tests/test_channels.py

```diff
@@ -747,3 +751,26 @@ def test_newsline_strip():
     message = UserMessage("\n/restart\n")
 
     assert message.text == "/restart"
+
+
+def test_register_channel_without_route():
+    """Check we properly connect the input channel blueprint if route is None"""
+    from rasa_core.channels import RestInput
+
+    # load your trained agent
+    agent = Agent.load(MODEL_PATH, interpreter=RegexInterpreter())
+
+    input_channel = RestInput()
+
+    from flask import Flask
+    import rasa_core
```

Comment:
Is this entire import necessary?

Reply:
yes, for `rasa_core.channels.channel.register`

Reply:
I was able to test this one locally by importing `channel` and then using `channel.register`

Reply:
yes sure, but that doesn't make any difference (ok it makes the line a little shorter where we call `register` but thats it :D - given that this is the way we call it in the other channels as well I don't see a reason to change this)